packages/babel-preset-react-app/create.js

```diff
@@ -30,6 +30,11 @@ module.exports = function(api, opts, env) {
   var isEnvTest = env === 'test';
 
   var isFlowEnabled = validateBoolOption('flow', opts.flow, true);
+  var isTypeScriptEnabled = validateBoolOption(
+    'typescript',
+    opts.typescript,
+    true
```

Comment:
I'm not much comfortable with this being `true` but it doesn't seem to cause any issue being turned on together with flow.

Reply:
The plugin is off by default unless it's applying against a `.tsx?` file, which is awesome. We should probably add some test case for this.